Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -51,7 +51,7 @@
 #define ENABLE_AIRPLAY_PICKER 1
 #endif
 
-#if !defined(ENABLE_ALTERNATE_WEBM_PLAYER) && (PLATFORM(MAC) || PLATFORM(IOS) || PLATFORM(MACCATALYST))
+#if !defined(ENABLE_ALTERNATE_WEBM_PLAYER) && !PLATFORM(MACCATALYST) && !PLATFORM(WATCHOS) && !PLATFORM(APPLETV)
```

Comment:
This was changed to address `PLATFORM(MACCATALYST)` not having `MEDIA_SOURCE` enabled.